A3.md

```diff
@@ -141,7 +151,7 @@ message TraceNode {
   // error assosiated with the channel event
```

Comment:
(This comment actually belongs a few lines above, but github won't let me put it there while looking at a specific commit.)

What happens when an individual trace node references a specific subchannel (for example, when adding or removing a subchannel to the parent channel)?  Don't we need some way to indicate which subchannel was affected?  Is that going to be indicated in the `data` field, or do we need some other way to reference the subchannel?